src/charm.py

```diff
@@ -153,147 +166,46 @@ def _patch_service(self, name: str, ro_port: int, rw_port: int) -> None:
             field_manager=self.model.app.name,
         )
 
-    def get_secret(self, scope: str, key: str) -> Optional[str]:
```

Comment:
we must keep get/set secret as a placeholder for the migration to juju-3.1 , where secrets support is implemented